### ISSUE TITLE
[$10]Members registration: Colour contrast issues

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3240]
**Describe the bug**

Members registration: Colour contrast issues

**To Reproduce**
1. Go to the Members registration page:https://accounts.topcoder.com/member/registration

**Expected behavior**

The colour contrast should comply as per the WCAG minimum contrast requirements.

**Actual behavior**
The colour contrast between the background and foreground of the elements doesn't comply with WCAG minimum contrast requirements.  Text Links: Text Link color for Terms, Privacy Policy, and Log in  do not comply with WCAG Standards

**Additional context**

Type: Color Contrast

Page: https://accounts.topcoder.com/member/registration

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099911 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099911 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099911 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by mishacucicea** [comment]
@nkumar-topcoder Can we merge this?

**Comment by nkumar-topcoder** [comment]
> @nkumar-topcoder Can we merge this?

@sushilshinde pls look into this

**Comment by drasticdpk** [comment]
@r0hit-gupta @mishacucicea  can you please confirm it ? still it is fail, here is the reference. 

![image](https://user-images.githubusercontent.com/25297859/64318911-e7c86180-cfda-11e9-95c5-90c305890a8b.png)

**Comment by drasticdpk** [comment]
Request to update Login text too.
![image](https://user-images.githubusercontent.com/25297859/64431083-6c99a500-d0d9-11e9-8500-375f24dc83a2.png)

**Comment by r0hit-gupta** [comment]
@drasticdpk everything is fine for me. It might be a deployment issue.

![image](https://user-images.githubusercontent.com/16841686/64451601-2e64ab80-d102-11e9-8756-f879d676fa21.png)

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099911<br/><br/>```This is an automated message for crazyk via Topcoder X```
